mindsdb/integrations/handlers/bigquery_handler/bigquery_handler.py

```diff
@@ -132,10 +149,18 @@ def get_columns(self, dataset, table_name) -> Response:
         'type': ARG_TYPE.STR,
         'description': 'The BigQuery project id.'
     },
+    dataset={
```

Comment:
Can we just include this in the README also as required param

Reply:
Updated. Also added service_account_json parameter